src/Inspector/utils/inspect.ts

```diff
@@ -28,13 +34,98 @@ export const getElementCodeInfo = (element: HTMLElement): CodeInfo | undefined =
     }
   }
 
+  return undefined
+}
+
+/**
+ * judge if the inspected element is a component
+ */
+const isComponentFiber = (fiber: Fiber) => {
+  if (fiber.index || fiber.sibling) {
```

Comment:
I still don't understand why need judge `fiber.index || fiber.sibling`

Reply:
for example

Reply:
but how about `const App = () => { return ( <> <App2></App2> <App3></App3> </> ) }`

Reply:
`
const App2 = <div></div>
const App3 = <span></span>

const App = () => {
  return (
    <>
       <App2 />
       <App3 />
    </>
  )
}
`
when we inspect the element, it will always get its actual dom node's fiber 
click App2 to get div's fiber and jump to App2's place of reference
`
// div's fiber
 {
  return : {} // App2's fiber
}
`
However if a component like App which root node is `<></>`, we can't jump to the place of reference.And after my test, the component like that can not be inspected at all.